src/components/AvatarRow/AvatarRow.jsx

```diff
@@ -0,0 +1,168 @@
+import React, { useRef, useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import throttle from 'lodash.throttle'
+import classNames from 'classnames'
+import { noop } from '../../utilities/other'
+import { AvatarRowUI, CounterAvatarUI } from './AvatarRow.css'
+import Avatar from '../Avatar'
+import Tooltip from '../Tooltip'
+import { config } from '../Avatar/Avatar.css'
+import { setupObserver, splitAvatarsArray } from './AvatarRow.utils'
+import { number } from '@storybook/addon-knobs'
```

Comment:
Should this import be here?

Reply:
Ha, I don't even know how that got there... I think VSCode's was being super "helpful" auto importing it...

Reply:
Yep, that's what I figured too!